src/components/Experiment/Footer.js

```diff
@@ -24,26 +23,30 @@ function Footer({
               </option>
             ))}
           </select>
-        </div>
-        <div className="ex-footer__actions">
-          <Button bsSize="large" bsStyle="success" onClick={markSimilar}>
-            Identical
-          </Button>
-          <Button bsSize="large" onClick={markMaybe}>
-            Similar
-          </Button>
-          <Button bsSize="large" bsStyle="danger" onClick={markDifferent}>
-            Different
-          </Button>
-        </div>
-        <div className="ex-footer__buttons">
-          <Button onClick={finish}>Finish</Button>
-          <Button bsStyle="info" onClick={skip}>
-            Skip
-          </Button>
-        </div>
-      </div>
-    </div>
+        </Col>
+        <Col xs={6} className="text-center">
+          <ButtonToolbar style={{ display: 'inline-block' }}>
```

Comment:
Why is this inline style really needed?

Reply:
I mean... why not doing it in the css side?

Reply:
the easiest way to center ButtonToolbar I found. Parent with `text-center` + `inline-block`.
any ideas?

Reply:
will move to css, no problem here